src/Sylius/Bundle/ShopBundle/Gulpfile.js

```diff
@@ -23,7 +23,7 @@ var paths = {
         ],
         sass: [
             '../UiBundle/Resources/private/sass/**',
```

Comment:
Will it work? With Sylius installed as single components (not `sylius/sylius` replacing all the other packages), the UiBundle will be at `vendor/sylius/ui-bundle/` and ShopBundle at `vendor/sylius/shop-bundle/`.